### DISCUSSION TITLE
Two users under same email

### DISCUSSION THREAD

**Discussion by KasteloShep** [dani-garcia/vaultwarden#3395] [Q&A]
Hello;
I seem to have an issue where I have two users that are under the same email. One was their original email, and a second is an invite. When I try to take action and delete the duplicate, it comes back as "wrong email". Any attempt to access the second account fails since it's redirected to the first "verified" account.

**Comment by BlackDex** [comment]
Do you have any screenshots or logs to show.
Are you sure it's the exact same address?
Can you manually access the database and see if there is a difference?

**Comment by BlackDex** [comment]
@Kastilo do you have more info? Else i need to close this, since no way for us to reproduce this. Also the column which stores the email has a unique on it, which makes it impossible to have multiple exactly the same records with that exact same email address.

**Comment by WhiteBlackGoose** [comment]
If someone hits this issue again: when you're prompted to input the email, try copying the prompt with quotes, then paste it into the text field, and carefully remove the quotes. The reason for this error may be due to invisible characters in the email, which are preserved if you copy it and remove the quotes, so it will delete the user.

That issue with invisible characters may also be the reason why it fails to register a new account with the correct email address.

In my case it might have been due to the fact that when I was inviting my colleagues, I copy-pasted their email addresses from a microsoft app which leaves those weird trailing characters, and looks like back then vaultwarden didn't trim the addresses. Make sure to sanitize the address yourself in any text editor which shows invisible characters.
